tests/system.rs

```diff
@@ -1285,29 +1285,37 @@ fn keeps_newlines_in_external_mod_doc() {
     assert_eq!("The mods multiline documentation\n\nwith an empty line", got.docs);
 }
 
+/// Addresses https://github.com/racer-rust/racer/issues/618
 #[test]
-fn issue_618() {
+fn always_get_all_doc_lines() {
```

Comment:
I moved the issue number to the doc comment so that I could include the full URL. I'm okay with putting issue numbers in test names for the purposes of `cargo test 618` but without easy reference to the issue I'm not sure how valuable the number is.